### DISCUSSION TITLE
Any way to tell BetterBibtex is ready?

### DISCUSSION THREAD

**Discussion by boan-anbo** [retorquere/zotero-better-bibtex#2084] [Q&A]
Hi! I'm writing a plugin that uses citation key. I was hoping to find a way to check if BetterBibtex has finished initiating and is ready to use (otherwise the `citationKey` field is empty). Which function should I call? Thanks!

**Comment by retorquere** [comment]
You can `await Zotero.BetterBibTeX.ready`. You can always request `item.getField('citationKey')`, but it will return an empty string until `ready` resolves.
